Sources/Networking/HTTPClient/HTTPClient.swift

```diff
@@ -222,7 +229,14 @@ private extension HTTPClient {
         var headers: HTTPClient.RequestHeaders
         var verificationMode: Signing.ResponseVerificationMode
         var completionHandler: HTTPClient.Completion<Data>?
-        var retried: Bool = false
+
+        /// Whether the request has been retried.
+        var retried: Bool {
+            return self.retryCount > 0
+        }
+
+        /// The number of times that we have retried the request
+        var retryCount: UInt = 0
```

Comment:
I'm using a `UInt` instead of an `Int` here so there won't be any negative values 😎